src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -2630,5 +2630,20 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_BadSKknown, "System").WithArguments("System", "namespace", "type").WithLocation(6, 17)
                 );
         }
+
+        [Fact]
+        public void DefaultNullableParameter()
+        {
+            var text = @"
+class C
+{
+    static void Main() => A();
+
+    static void A(int? x = default) => System.Console.WriteLine(x?.ToString() ?? ""null"");
```

Comment:
Test would be more straightforward with `System.Console.WriteLine(x.HasValue)` which would print "False".
Can you also print the result for invoking `B(int? x = default(int?))`?